arch/arm64/boot/dts/freescale/imx8mp-maivin.dts

```diff
@@ -181,6 +181,25 @@
 			MX8MP_IOMUXC_SAI1_RXC__GPIO4_IO01		0x184	/* SODIMM 222 */
 		>;
 	};
+
+	/*Pins used to control the power to the modem*/
```

Comment:
Missing spaces after /* and before */.

Reply:
I ran that commit through checkpath and it said there was no problems
Is that a personal preference?